locale/en/LC_MESSAGES/django.po

```diff
@@ -32,478 +32,259 @@ msgid ""
 "%(street_address_2)s"
 msgstr ""
 
-#: templates/templated_email/account/account_delete.email:4
-msgctxt "Account delete reset e-mail subject"
-msgid "Delete your account"
+#: templates/read_only/read_only_splash.html:48
```

Comment:
Email translations should come from `email` and `mjml` files and not the compiled HTML ones. The right command to export new translations is:
```
python manage.py makemessages -l en --extension=email,html,mjml,py,txt --ignore="templates/templated_email/compiled" --ignore="node_modules"
```